src/charm.py

```diff
@@ -36,10 +37,10 @@ def __init__(self, *args):
         self.framework.observe(self.on.prometheus_pebble_ready, self._on_pebble_ready)
         self.framework.observe(self.on.config_changed, self._on_config_changed)
         self.framework.observe(self.on.stop, self._on_stop)
-        self.framework.observe(self.on['alertmanager'].relation_changed,
-                               self._on_alertmanager_changed)
-        self.framework.observe(self.on['alertmanager'].relation_broken,
-                               self._on_alertmanager_broken)
+        # self.framework.observe(self.on['alertmanager'].relation_changed,
```

Comment:
Why not just delete it instead of commenting it out ? Or are you planning to uncomment it in a future revision of this draft ?

Reply:
Removed